src/components/admin/ReportView.tsx

```diff
@@ -35,9 +39,39 @@ export const ReportView: React.FC<Props> = ({
   type,
   associatedObject,
   key,
+  assignee,
+  adminList,
+  reportId,
 }) => {
   const { isOpen, onOpen, onClose } = useDisclosure();
 
+  const assignedAdmin = useRef<HTMLSelectElement>(null);
+
+  const toast = useToast();
+
+  const navigate = useNavigate();
+
+  const save = async () => {
+    if (assignedAdmin.current === undefined) {
+      return;
+    }
+
+    try {
+      await assignAdmin(reportId, parseInt(assignedAdmin.current!.value));
+      navigate(0);
```

Comment:
this just reloads right? is there some benefit to this vs. using `window.location.reload()` (or whatever it is)? perhaps it keeps the react-router history? not sure but would love to know

Reply:
`navigate(0)` only reloads the current route, whereas `window.location.reload()` also reloads the assets. `navigate(0)` is more react-friendly and has less overhead.